src/libs/OptionsListUtils/index.ts

```diff
@@ -913,6 +915,14 @@ function createOption(
                 ? personalDetail.login
                 : getAlternateText(result, {showChatPreviewLine, forcePolicyNamePreview}, !!result.private_isArchived, lastActorDetails);
         reportName = showPersonalDetails ? getDisplayNameForParticipant({accountID: accountIDs.at(0)}) || formatPhoneNumber(personalDetail?.login ?? '') : getReportName(report);
+
+        const oneTransactionThreadReportID =
+            report.type === CONST.REPORT.TYPE.IOU || report.type === CONST.REPORT.TYPE.EXPENSE || report.type === CONST.REPORT.TYPE.INVOICE
+                ? getOneTransactionThreadReportID(report, chatReport, allSortedReportActions[report.reportID])
+                : undefined;
+        const oneTransactionThreadReport = oneTransactionThreadReportID ? allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${oneTransactionThreadReportID}`] : undefined;
+
+        result.isUnread = isUnread(report, oneTransactionThreadReport, !!result.private_isArchived) && !!report.lastActorAccountID;
```

Comment:
This is a global function used in many places. Can you apply these chagnes to only Search list view.

Reply:
It's still generic. Can you please carefully check the code and apply the changes to the correct places? I think the correct function is `getSearchOptions`. Check this and test it. Make sure that other options' functions' behaviour does not change. 

Also, test other pages where these options functions are used, they should behave the same.

Reply:
Also these search options are used at couple places so make sure that those are not changed unless we should change them too.